### DISCUSSION TITLE
R Studio

### DISCUSSION THREAD

**Discussion by giscus** [AwaitQuality/windowsonarm#52] [General]
# R Studio

Check if R Studio is ARM ready on Windows and can be used on Windows ARM devices such as the Surface with the Snapdragon X Elite.

https://windowsonarm.org/8fdbdfd0-a352-45dd-ae28-1959d4f2017a

**Comment by christopheragnus** [comment]
This app works perfectly in Prism with R 4.4.1. There's also a R tools binary available for ARM but you need to rename the folder to C:/rtools44

**Reply by OpenSource03** [reply]
Updated, thanks.

**Comment by talynone** [comment]
There's an open issue for Quarto documents:

https://github.com/rstudio/rstudio/issues/15277

The work-around is to run the Linux ARM version of R under WSL:

https://www.linkedin.com/pulse/running-rstudio-arm-based-windows-pcs-wsl-ftw-andrew-clinick-zcxdc/
